package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "node-red-contrib-rhea",
-  "version": "0.0.2",
+  "version": "0.0.3",
```

Comment:
did you try if with newer rhea version all the other stuff works or maybe there are some breaking changes ?

Reply:
a long time ago, without any pb

Reply:
not yet, I was off during May :-(

Reply:
Ah ok ... so just fix the conflicts and then I'll wait for some test before merging.

Reply:
tell me if I am wrong but I didn't change the rhea version, 
0.0.3 is node-red-contrib-rhea module version

Reply:
damn you are right! So can you just resolve conflicts in order to merge this PR?Thanks!